excalidraw-app/collab/Collab.tsx

```diff
@@ -801,41 +804,39 @@ class Collab extends PureComponent<Props, CollabState> {
     CURSOR_SYNC_TIMEOUT,
   );
 
-  onScrollAndZoomChange = throttle(
-    (payload: { zoom: AppState["zoom"]; scroll: { x: number; y: number } }) => {
-      const appState = this.excalidrawAPI.getAppState();
-
-      if (appState.amIBeingFollowed) {
-        const { x: x1, y: y1 } = viewportCoordsToSceneCoords(
-          { clientX: 0, clientY: 0 },
-          {
-            offsetLeft: appState.offsetLeft,
-            offsetTop: appState.offsetTop,
-            scrollX: payload.scroll.x,
-            scrollY: payload.scroll.y,
-            zoom: payload.zoom,
-          },
-        );
+  relaySceneBounds = throttle((props?: { shouldPerform: boolean }) => {
```

Comment:
this should have used our `throttleRAF` I assume?

Reply:
Yes, good catch, thank you!